include.js

```diff
@@ -1261,6 +1261,14 @@ let cards = [
     author: 'Divyansh',
     githubLink: 'https://github.com/okaydivyansh'
   },
+
+  {
+    artName: 'wheel', // change this to the name of your artwork
+    pageLink: './Art/mohdUsman-6508/triangle.html', // change this
```

Comment:
The file name is incorrect, it needs to be named after the file name of _your_ contribution, so it's `wheel.html`.

Reply:
i have chaged it.